pkgs/by-name/kn/knossosnet/package.nix

```
@@ -0,0 +1,34 @@
+{
+  buildDotnetModule
+, fetchFromGitHub
+, fontconfig
+, lib
+, openal
+, stdenv
+, xorg
+}:
+
+let
+  version = "0.2.0-RC10";
+  executable = "Knossos.NET";
+in
+buildDotnetModule {
+  pname = "knossosnet";
+  inherit version;
+  nugetDeps = ./deps.nix;
+  executables = [ executable ];
+  patches = [ ./targetframework.patch ];
+  runtimeDeps = [ fontconfig openal xorg.libX11 xorg.libICE xorg.libSM ];
+  src = fetchFromGitHub {
+    owner = "KnossosNET";
+    repo = "Knossos.NET";
+    rev = "v${version}";
+    hash = "sha256-nMxi6OPoyXrrjPa60DM7ZSCQ/zOuwWNuDm0rb7oO3Yc=";
+  };
```

a more standard formatting could look like:

```suggestion
buildDotnetModule rec {
  pname = "knossosnet";
  version = "0.2.0-RC10";

  src = fetchFromGitHub {
    owner = "KnossosNET";
    repo = "Knossos.NET";
    rev = "v${version}";
    hash = "sha256-nMxi6OPoyXrrjPa60DM7ZSCQ/zOuwWNuDm0rb7oO3Yc=";
  };

  patches = [ ./targetframework.patch ];

  nugetDeps = ./deps.nix;
  executables = [ "Knossos.NET" ];

  runtimeDeps = [ fontconfig openal xorg.libX11 xorg.libICE xorg.libSM ];
```